packages/nuxt/index.cjs

```diff
@@ -0,0 +1,6 @@
+// CommonJS proxy to bypass jiti transforms from nuxt 2 and using native ESM
+module.exports = function (...args) {
```

Comment:
should use `exports.default`, you can run `pnpm pack` and test the tgz here: https://arethetypeswrong.github.io/

mlly will check it